kafka_scanner/msg_processor.py

```diff
@@ -2,9 +2,9 @@
 
 class MsgProcessor(object):
 
-    def __init__(self, handlers_list, encoding=None):
+    def __init__(self, handlers_list, encoding=None, *args, **kwargs):
         self._handlers = []
-        self.processor_handlers = MsgProcessorHandlers(encoding=encoding)
+        self.processor_handlers = MsgProcessorHandlers(encoding=encoding, *args, **kwargs)
```

Comment:
it is weird that positional arguments *args come after a key argument encoding=. Didn't pylint complain here?

Reply:
no, afaik this is perfectly valid. If it goes against SH style guide let me know and I can make it an unnamed argument (i always prefer using names for the shake of documenting but I see your point).